feat: translate best practices page (#85)
monterail/vue3-docs-pl: src/guide/best-practices/security.md

</a>
```

> Załóżmy, że `sanitizedUrl` został wyczyszczony, tak aby był to z pewnością prawdziwy adres URL, a nie JavaScript. Dzięki `userProvidedStyles` złośliwi użytkownicy mogliby nadal udostępniać CSS, aby „kliknąć jacka”, np. stylizując łącze w przezroczyste pole nad przyciskiem „Zaloguj się”. Następnie, jeśli `https://user-controlled-website.com/` jest zbudowany tak, aby przypominał stronę logowania Twojej aplikacji, mogliby po prostu przechwycić prawdziwe dane logowania użytkownika.

Reviewer: ```suggestion
Załóżmy, że `sanitizedUrl` został wyczyszczony, tak aby był to z pewnością prawdziwy adres URL, a nie JavaScript. Dzięki `userProvidedStyles` złośliwi użytkownicy mogliby nadal udostępniać CSS, aby „przechwytywać kliknięcia”, np. stylizując łącze w przezroczyste pole nad przyciskiem „Zaloguj się”. Następnie, jeśli `https://user-controlled-website.com/` jest zbudowany tak, aby przypominał stronę logowania Twojej aplikacji, mogliby po prostu przechwycić prawdziwe dane logowania użytkownika.
```

## Dobre praktyki {#best-practices}

Ogólna zasada jest taka, że ​​jeśli pozwolisz na wykonywanie nieoczyszczonej, dostarczonej przez użytkownika treści (jako HTML, JavaScript, a nawet CSS), możesz narazić się na ataki. Ta rada jest prawdziwa niezależnie od tego, czy używasz Vue, innego frameworka, czy nawet żadnego frameworka.

Poza zaleceniami podanymi powyżej dla [Potencjalnych zagrożeń](#potential-dangers), zalecamy również zapoznanie się z tymi zasobami:

- [Arkusz informacyjny dotyczący bezpieczeństwa HTML5](https://html5sec.org/)
- [Arkusz informacyjny OWASP dotyczący zapobiegania atakom typu Cross Site Scripting (XSS)](https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html)

Następnie wykorzystaj zdobytą wiedzę, aby przejrzeć kod źródłowy swoich zależności pod kątem potencjalnie niebezpiecznych wzorców, jeśli którykolwiek z nich zawiera komponenty stron trzecich lub w inny sposób wpływa na to, co jest renderowane w DOM.

> ## Koordynacja zaplecza {#backend-coordination}

Reviewer: ```suggestion
## Koordynacja z backendem {#backend-coordination}
```

> Luki w zabezpieczeniach HTTP, takie jak fałszowanie żądań między witrynami (CSRF/XSRF) i dołączanie skryptów między witrynami (XSSI), są rozwiązywane głównie w zapleczu, więc nie są problemem Vue. Niemniej jednak nadal dobrym pomysłem jest komunikacja z zespołem zaplecza, aby dowiedzieć się, jak najlepiej współdziałać z ich API, np. przesyłając tokeny CSRF wraz z formularzami.

Reviewer: ```suggestion
Luki w zabezpieczeniach HTTP, takie jak fałszowanie żądań między witrynami (CSRF/XSRF) i dołączanie skryptów między witrynami (XSSI), są rozwiązywane głównie w backendzie, więc nie są problemem Vue. Niemniej jednak nadal dobrym pomysłem jest komunikacja z zespołem backendowym, aby dowiedzieć się, jak najlepiej współdziałać z ich API, np. przesyłając tokeny CSRF wraz z formularzami.
```
